### PR TITLE
complete class-constructor-new

### PR DESCRIPTION
**Pull request by CuongNgMan**

### CODE REVIEW

File: class-constructor-new/exercise-1.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   * @param {Vector} ve
+   * @returns {boolean}
+   */
+  equals(ve) {
+    if (!ve || !(ve instanceof Array)) {
+      throw new Error('Not a valid Vector');
+    }
+
+    return this.length === ve.length;
+  }
```
Comment (viktordavidovich) [inline class-constructor-new/exercise-1.js]: I believe we need to compare not only vectors length, but "each component" like mentioned in the task 

// from the task
// an equals method, to check that two vectors that have the **same components are equal**

Comment (viktordavidovich) [reply class-constructor-new/exercise-1.js]: I noticed that you're using this method in other places to check whether two vectors have the same length, so please, don't forget to update such places.

File: package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
   "homepage": "https://github.com/CuongNgMan/Mentoring-HTML-CSS#readme",
   "devDependencies": {
     "@babel/core": "^7.16.0",
+    "@babel/node": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
     "babel-jest": "^27.3.1",
```
Comment (viktordavidovich) [inline package.json:28]: helmet packages has not been added into devDependecies :) 
I had to manually install it to run tests. 
Please, don't forget adding all necessary dependencies into package.json

**Review by viktordavidovich — APPROVED** [review]
Please, review comments.
Good job!

File: class-constructor-new/exercise-1.js [hunk 3]
```diff
@@ -20,15 +21,16 @@ export class Vector extends Array {
    * @returns {boolean}
    */
   equals(ve) {
-    if (!ve || !(ve instanceof Array)) {
-      throw new Error('Not a valid Vector');
-    }
-
-    return this.length === ve.length;
+    return this.isHavingSameLength(ve) && this.every((c, i) => c === ve[i])
```
Comment (CuongNgMan) [inline class-constructor-new/exercise-1.js:24]: @viktordavidovich equals function compares each value of both vectors before yielding result.

**Pull request merged**